api/kyverno/v2/policy_exception_types.go

```diff
@@ -159,3 +212,76 @@ type PolicyExceptionList struct {
+type ExceptionReportMode string
+
+const (
+	// ExceptionReportSkip skips the rule entirely (default behavior)
+	ExceptionReportSkip ExceptionReportMode = "skip"
+	// ExceptionReportWarn generates a warning but allows the resource
+	ExceptionReportWarn ExceptionReportMode = "warn"
+	// ExceptionReportPass reports the rule as passed
+	ExceptionReportPass ExceptionReportMode = "pass"
+)
+
+// ImageException specifies image-based exception criteria
+type ImageException struct {
+	// ImageReferences is a list of image reference patterns to be exempted.
+	// Wildcards ('*' and '?') are allowed. See: https://kubernetes.io/docs/concepts/containers/images.
+	ImageReferences []string `json:"imageReferences"`
+}
+
+// Validate implements programmatic validation for ImageException
+func (i *ImageException) Validate(path *field.Path) (errs field.ErrorList) {
+	if len(i.ImageReferences) == 0 {
+		errs = append(errs, field.Required(path.Child("imageReferences"), "Image exception requires at least one image reference"))
+	}
+	return errs
+}
+
+// ValueException specifies value-based exception criteria
+type ValueException struct {
+	// Path is a JSONPath expression that identifies the field to check for exempted values.
+	// For example: "spec.containers[*].securityContext.runAsUser" or "metadata.labels.environment"
+	Path string `json:"path"`
+
+	// Values is a list of values that should be exempted from validation.
+	Values []string `json:"values"`
+
+	// Operator specifies how values should be matched against the exempted values.
+	// Supported operators: "equals" (default), "in", "startsWith", "endsWith", "contains"
+	// +optional
+	// +kubebuilder:default="equals"
+	// +kubebuilder:validation:Enum=equals;in;startsWith;endsWith;contains
+	Operator *ValueOperator `json:"operator,omitempty"`
+}
+
+// Validate implements programmatic validation for ValueException
+func (v *ValueException) Validate(path *field.Path) (errs field.ErrorList) {
+	if v.Path == "" {
+		errs = append(errs, field.Required(path.Child("path"), "Value exception requires a path"))
+	}
+	if len(v.Values) == 0 {
+		errs = append(errs, field.Required(path.Child("values"), "Value exception requires at least one value"))
+	}
+	return errs
+}
+
+// ValueOperator specifies how values should be matched
+// +kubebuilder:validation:Enum=equals;in;startsWith;endsWith;contains
+type ValueOperator string
+
+const (
+	// ValueOperatorEquals matches exact values
+	ValueOperatorEquals ValueOperator = "equals"
+	// ValueOperatorIn checks if value is in the list
+	ValueOperatorIn ValueOperator = "in"
+	// ValueOperatorStartsWith checks if value starts with any of the exempted values
+	ValueOperatorStartsWith ValueOperator = "startsWith"
+	// ValueOperatorEndsWith checks if value ends with any of the exempted values
+	ValueOperatorEndsWith ValueOperator = "endsWith"
+	// ValueOperatorContains checks if value contains any of the exempted values
+	ValueOperatorContains ValueOperator = "contains"
+)
```

Comment:
If we agreed that we need ValueException, I think we should use the [Condition](https://github.com/kyverno/kyverno/blob/8b26a1ce076d80ec128fd7ff011acc7e6d370174/api/kyverno/v1/common_types.go#L282) struct